scipy/stats/morestats.py

```diff
@@ -2132,21 +2202,39 @@ def ansari(x, y):
     are different.
 
     >>> ansari(x1, x2)
-    AnsariResult(statistic=511.0, pvalue=0.35506083719834347)
+    AnsariResult(statistic=511.0, pvalue=0.3550608509212331)
```

Comment:
As we are removing seed from examples, specify that the result is random so the test passes.
```suggestion
    AnsariResult(statistic=511.0, pvalue=0.3550608509212331)  # random
```

Reply:
> As we are removing seed from examples, specify that the result is random so the test passes.

I don't think we use a seed anywhere. There is no random variate generation.